src/dispatchers/mouseDispatcher.ts

```diff
@@ -2,110 +2,119 @@
+      svg.addEventListener("mousemove", (e) => this._processMoveEvent(e));
+      svg.addEventListener("mouseout", (e) => this._processMoveEvent(e));
     }
 
     /**
-     * Gets the current callback to be called on mousemove.
+     * Registers a callback to be called whenever the mouse position changes,
+     * or removes the callback if `null` is passed as the callback.
      *
-     * @return {(location: Point) => any} The current mousemove callback.
+     * @param {any} key The key associated with the callback.
+     *                  Key uniqueness is determined by deep equality.
+     * @param {(p: Point) => any} callback A callback that takes the pixel position
+     *                                     in svg-coordinate-space. Pass `null`
+     *                                     to remove a callback.
+     * @return {Dispatcher.Mouse} The calling Dispatcher.Mouse.
      */
-    public mousemove(): (location: Point) => any;
-    /**
-     * Attaches a callback to be called on mousemove.
-     *
-     * @param {(location: Point) => any} callback A function that takes the pixel position of the mouse event.
-     *                                            Pass in null to remove the callback.
-     * @return {Mouse} The calling Mouse Handler.
-     */
-    public mousemove(callback: (location: Point) => any): Mouse;
-    public mousemove(callback?: (location: Point) => any): any {
-      if (callback === undefined) {
-        return this._mousemove;
+    public onMouseMove(key: any, callback: (p: Point) => any): Mouse {
+      if (callback === null) { // remove listener if callback is null
+        this.broadcaster.deregisterListener(key);
+      } else {
+        this.broadcaster.registerListener(key, () => { callback(this.getLastMousePosition()); });
       }
-      this._mousemove = callback;
       return this;
     }
 
+    private _processMoveEvent(e: MouseEvent) {
+      this._computeMousePosition(e.clientX, e.clientY);
+      this.broadcaster.broadcast();
+    }
+
     /**
-     * Gets the current callback to be called on mouseout.
-     *
-     * @return {(location: Point) => any} The current mouseout callback.
+     * Computes the mouse position relative to the <svg> in svg-coordinate-space.
      */
-    public mouseout(): (location: Point) => any;
+    private _computeMousePosition(clientX: number, clientY: number) {
+      // get the origin
+      this._measureRect.setAttribute("x", "0");
+      this._measureRect.setAttribute("y", "0");
+      var mrBCR = this._measureRect.getBoundingClientRect();
+      var origin = { x: mrBCR.left, y: mrBCR.top };
+
+      // caculate the scale
+      this._measureRect.setAttribute("x", "100");
+      this._measureRect.setAttribute("y", "100");
+      mrBCR = this._measureRect.getBoundingClientRect();
+      var testPoint = { x: mrBCR.left, y: mrBCR.top };
+
+      var scaleX = (testPoint.x - origin.x) / 100;
+      var scaleY = (testPoint.y - origin.y) / 100;
```

Comment:
Scale portion for CSS zoom.  Any number would work as well as 100, so long as it's set as the attribute and used to measure. Using a larger integer should avoid floading-point problems.